manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -61,7 +61,7 @@ spec:
 {{- if or .Values.cni.pullPolicy .Values.global.imagePullPolicy }}
           imagePullPolicy: {{ .Values.cni.pullPolicy | default .Values.global.imagePullPolicy }}
 {{- end }}
-          command: ["/install-cni.sh"]
+          command: ["/usr/local/bin/install-cni"]
```

Comment:
nit: /usr/local/bin/install-cni -> install-cni

No need to pin it to a specific path as far as I know?

Reply:
Ah right because it should already be in PATH. I changed it to `install-cni`